rudder-web/src/main/resources/logback.xml

```diff
@@ -164,103 +165,240 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
   -->
   <logger name="trace.ldif.in.file" level="off" />
 
+  <!--
+    Debug LDAP queries related to inventory search
+    ==============================================
+    Uncomment and set to "debug" or "trace" to have precise logs about
+    LDAP queries done by the node search engine
+  -->
+  <logger name="com.normation.rudder.services.queries" level="off" />
 
-  <!-- 
+  <!--
     Dump Node Configurations
     ========================
     See information in the "rudder.debug.nodeconfiguration.path"
     application property.
-    
-    To enable that debug, set level to "debug". 
+
+    To enable that debug, set level to "debug".
     To disable that debug, set level to "off".
    -->
   <logger name="rudder.debug.nodeconfiguration" level="off" />
 
-  
-  <!-- ==================================================== -->
-  <!-- YOU SHOULD NOT HAVE TO CHANGE THINGS BELOW THAT LINE -->
-  <!-- ==================================================== -->
-  
-  <!-- 
-    Display AJAX information of the Web interface
-    =============================================
-    Whatever the root logger level is, you are likely
-    to not wanting this information. 
-    Set the level to debug if you are really interested
-    in AJAX-related debug messages.
+
+  <!--
+    New node registration
+    =====================
+    Set level to debug or trace to have more precise information
+    about the different acceptation steps of nodes.
+   -->
+  <logger name="com.normation.rudder.services.servers.NewNodeManagerImpl" level="info" />
+
+
+  <!--
+    Plugins
+    =======
+    Set level to debug or trace to have more information about plugin
+    registration and information.
+   -->
+  <logger name="com.normation.rudder.plugins" level="info" />
+
+
+  <!--
+     Promise generation and writing
+     ==============================
+     The following logger allow to see information about
+     the promise generation process, like step of generation and
+     their timing, information about writing files, etc.
+   -->
+   <!--
+     Set to info to only have macro information, debug to have timing information about
+     each step (good default), trace to have more information.
+   -->
+   <!-- Rudder 3.1 and below -->
+  <logger name="com.normation.rudder.services.policies.DeploymentServiceImpl" level="debug" />
```

Comment:
More on that... Debug should not be the "good default". I think it makes sense to always have timing information. Also, I think they should get their log file, in a line-by-line json format (json and not CSV so that we can add/remove inter-point with the level of debug), so that we can easely add it in the anon stats.